### ISSUE TITLE
development board model used

### ISSUE THREAD

**Issue by Zgb5566113** [Gabalo/RO_PUF#1]
Hello, thanks for sharing the code, may I ask the specific model of the development board used in the project?

**Comment by Gabalo** [comment]
I used a Nexys A7

**Comment by Zgb5566113** [comment]
Thank you for answer, Xilinx Artix-7 FPGA has many models, such as XC7A200T and XC7A100T. Different models have different hardware resource content. May I ask which specific MODEL of FPGA you use?

**Comment by Zgb5566113** [comment]
Hello, the model of the development board is very important to me, please reply when you see it, I would be very grateful

**Comment by Zgb5566113** [comment]
Are you using Digilent Nexys 4-100T Aritil-7 CSG324?

**Comment by Gabalo** [comment]
It was a 100T yeah

**Comment by Zgb5566113** [comment]
thank you very much!

**Comment by Zgb5566113** [comment]
Hello, thanks for your previous reply. According to your instructions, I adopted Xilinx Nexys 4-100T Aritil-7 CSG324 FPGA for comprehensive implementation in Vivado, but the number of IO ports would exceed the resources of development office. Therefore, I used Xilinx Nexys 4-200T Aritil-7 development board to successfully burn ro-PUF, and received data from FPGA through Matlab. What I am confused about is the intense input
